File: 0088-merge-sorted-array/Solution.js
var merge = function (nums1, m, nums2, n) {
    if (n > 0) {
        for (let x = 0; x < n; x++) {
            nums1[x + m] = nums2[x];
        }
    }

    nums1.sort(function (a, b) { return a - b });
};

File: 0088-merge-sorted-array/Solution.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        if (n > 0) {
            for (int x = 0; x < n; x++) {
                nums1[x + m] = nums2[x];
            }
        }
        
        Arrays.sort(nums1);
    }
}
